include/deal.II/fe/fe_interface_values.h

```diff
@@ -497,8 +862,11 @@ FEInterfaceValues<dim, spacedim>::FEInterfaceValues(
   , internal_fe_subface_values_neighbor(mapping, fe, quadrature, update_flags)
   , fe_face_values(nullptr)
   , fe_face_values_neighbor(nullptr)
+  , fe_interface_views_cache(*this)
```

Comment:
I don't understand what you mean by this, can you clarify please?

Reply:
I am not quite clear what the purpose of `fe_interface_views_cache` is. Why do we need `fe_interface_views_cache`, `fe_face_values` and `fe_face_values_neighbor`? How are they related?

Reply:
It stores the ``FEInterfaceViews`` objects. 

> Why do we need fe_interface_views_cache

What else should operator[] return? I would think always returning a new temporary is not as efficient or what are you proposing?

> How are they related?

I don't understand your question, I am sorry.

Reply:
What @masterleinad is pointing out is that in contrast to `FEValues`, the `FEInterfaceValues` views are really cheap: They only store a pointer and a single integer. There is essentially no cost associated with creating these objects. So there doesn't seem to be much of a case for keeping a cache around for these objects because the cache does not actually cache any pre-computed information.